components/StyledTooltip.js

```diff
@@ -151,6 +151,8 @@ class StyledTooltip extends React.Component {
     containerLineHeight: PropTypes.string,
     containerCursor: PropTypes.string,
     delayHide: PropTypes.number,
+    /** If true, children will be rendered directly, without any tooltip. Useful to build conditional tooltips */
+    noTooltip: PropTypes.bool,
```

Comment:
A bit of refactor on this file, but the change is only about this option to skip the tooltip conditionally